### PR TITLE
feat: itemlogin schema state

### PR DESCRIPTION
**Pull request by Defferrard**

### CODE REVIEW

File: src/itemLogin/itemLogin.ts [hunk 1]
```diff
@@ -9,10 +9,17 @@ export enum ItemLoginSchemaType {
   AnonymousAndPassword = 'anonymous+password',
 }
 
+export enum ItemLoginSchemaState {
+  Active = 'active', // Guests can sign in and log in
+  Disabled = 'disabled', // Guests can't sign in or log in
+  Freeze = 'freeze', // Guests can only log in
```
Comment (pyphilia) [inline src/itemLogin/itemLogin.ts:15]: Sign in means log in. 
I think the pairs are: sign in/sign up/sign out - login/register/logout
```suggestion
  Active = 'active', // Guests can register and log in
  Disabled = 'disabled', // Guests can't register or log in
  Freeze = 'freeze', // Guests can only log in
```

File: src/itemLogin/factory.ts [hunk 2]
```diff
@@ -7,6 +11,7 @@ export function ItemLoginSchemaFactory(
   return {
     id: faker.string.uuid(),
     type: ItemLoginSchemaType.Username,
+    state: ItemLoginSchemaState.Active,
```
Comment (pyphilia) [inline src/itemLogin/factory.ts:14]: I think I prefer status.
```suggestion
    status: ItemLoginSchemaStatus.Active,
```

**Review by pyphilia — CHANGES_REQUESTED** [review]

**Review by ReidyT — COMMENTED** [review]
I think I prefer `status` too.

File: src/itemLogin/itemLogin.ts [hunk 3]
```diff
@@ -9,10 +9,17 @@ export enum ItemLoginSchemaType {
   AnonymousAndPassword = 'anonymous+password',
 }
 
+export enum ItemLoginSchemaState {
```
Comment (pyphilia) [inline src/itemLogin/itemLogin.ts:12]:
```suggestion
export enum ItemLoginSchemaStatus {
```

**Review by pyphilia — CHANGES_REQUESTED** [review]
Should this be status too?

**Review by pyphilia — APPROVED** [review]

**Pull request merged**